.travis.yml

```diff
@@ -25,7 +25,7 @@ env:
 before_script:
   - env | sort
   - export IMAGE="claranet/spryker-base"
-  - export BASE_IMAGE_TAG="${VARIANT:+-$VARIANT}"
+  - export BASE_IMAGE_TAG="${VARIANT:+$VARIANT}"
```

Comment:
hmm, I would expect this to be `${VARIANT:-VARIANT}` instead; or am I missing something here?

Reply:
It can actually be shortened to just `$VARIANT`. It seems this was done for cases where VARIANT is not set which should be never. It's part of the first version of this file and was kept ever since.